contracts/src/L1/L1UnlockedMigrationController.sol

```diff
@@ -134,14 +134,15 @@ contract L1UnlockedMigrationController is IERC1155Receiver, IERC721Receiver, ERC
             revert TokenIdMismatch(tokenId, expectedTokenId);
         }
         
-        // send migration data to L2
-        bytes memory dnsEncodedName = NameUtils.dnsEncodeEthLabel(migrationData.transferData.label);
-        bytes memory message = BridgeEncoder.encodeMigration(dnsEncodedName, migrationData);
-        bridge.sendMessage(message);
-
-        // if migrated to L1 then also setup the name on the L1
+        // if migrated to L1 then setup the name on the L1
         if (migrationData.toL1) {
             l1EjectionController.completeEjectionFromL2(migrationData.transferData);
+        } 
```

Comment:
Is this going to work, given the addition of `onlyBridge`?

Reply:
Good catch. Looks like the tests weren't doing their job properly. I've now replaced this wholesale with a role-based access control system using EAC.